src/libraries/System.Reflection/tests/TypeInfoTests.cs

```diff
@@ -490,6 +490,12 @@ public void IsEnumDefined_Invalid()
         [InlineData(typeof(CompoundClass4<string>), new Type[] { typeof(GenericInterface1<string>), typeof(TI_NonGenericInterface1) })]
         public void ImplementedInterfaces(Type type, Type[] expected)
         {
+            if (PlatformDetection.IsMonoRuntime)
```

Comment:
I believe the proper way to exclude this is to open an issue on Github with the specifics and then apply the ActiveIssue attribute to the method? That file has another example.

Reply:
Sounds good

Reply:
We can remove this condition since ActiveIssue is added?

Reply:
I think this has been fixed in #40632